src/component/header/Header.js

```diff
@@ -1,30 +1,41 @@
 import PropTypes from 'prop-types';
-
 import {FaSearch} from "react-icons/fa";
 
 import * as hs from './HeaderStyle';
-import {HeaderWrapper, ListSelection} from "./HeaderStyle";
+import OptionList from "../optionList/OptionList";
+import {useCallback, useState} from "react";
 
 function Header({searchOption}) {
 
+  const [searchOptions, setSearchOptions] = useState({
+    startDate: '',
+    endDate: '',
+    status: '',
+  });
+
+  const handleChange = useCallback((name, {target: {value}}) => {
+    setSearchOptions({...searchOptions, [name]: value})
+  }, [searchOptions]);
```

Comment:
```suggestion
  const handleChange = useCallback((name, value) => {
    setSearchOptions({...searchOptions, [name]: value})
  }, [searchOptions]);
```
값을 바로 주는게 가독성도 좋고 더 안전하겠군요

Reply:
ㅇㄱㄹㅇ

Reply:
호오! 고마워!